backbone.js

```diff
@@ -613,43 +629,48 @@
 
       options = _.extend({validate: true, parse: true}, options);
       var wait = options.wait;
+      var success = options.success || _.identity;
+      var error = options.error;
+      delete options.success;
+      delete options.error;
```

Comment:
Why do you care to delete these properties? It shouldn't matter if they stick around, no?